memcached-operator/Gopkg.lock

```diff
@@ -401,6 +401,19 @@
   revision = "61e0c23e9d2e217f8d95ac104a8f2545c102b5c3"
   version = "v0.6.0"
 
+[[projects]]
+  branch = "master"
+  digest = "1:8df91d97d584d4260ea25953715753ff5a8b93c46a40cef4f40350cb84d49a56"
+  name = "github.com/operator-framework/operator-sdk-samples"
```

Comment:
Yep it was added when I executed `dep ensure`

Reply:
Works fine for me. It compiles just fine without that, not sure why that was added by dep ensure, because when I did dep ensure on your branch it didn't add it for me. What version of dep are you using?

https://github.com/operator-framework/operator-sdk-samples/pull/67/files#diff-7841188f35bab27d8b4ae66913fe54bcR65 if you click on that, it resolves it just fine for me.

Reply:
Hum, it is strange. So, for you all is fine. 

```
$ dep version
dep:
 version     : v0.5.3
 build date  : 2019-05-19
 git hash    : 192eb44
 go version  : go1.12.5
 go compiler : gc
 platform    : darwin/amd64
 features    : ImportDuringSolve=false
```

Reply:
I have dep `version     : v0.5.4` no idea if that will fix it, maybe try updating. 

Yes when I do `dep ensure` on your branch and then `go build` nothing changes, no new dep gets added.

Reply:
IMHO: we can move forward without adding the vendor and check it in another PR as well too. Realy tks for all your help and support @LiliC

Reply:
Hi @LiliC I will try downgrade the dep version and I will let you know.

Reply:
The same occurs with the upper version as follows. 

```shell
camilamacedo@Camilas-MacBook-Pro ~/go/src/operator-sdk-samples/memcached-operator (ISSUE_1464) $ dep check
# Gopkg.lock is out of sync:
github.com/operator-framework/operator-sdk-samples/memcached-operator/pkg/apis: imported or required, but missing from Gopkg.lock's input-imports
github.com/operator-framework/operator-sdk-samples/memcached-operator/pkg/apis/cache/v1alpha1: imported or required, but missing from Gopkg.lock's input-imports
github.com/operator-framework/operator-sdk-samples/memcached-operator/pkg/controller: imported or required, but missing from Gopkg.lock's input-imports
github.com/operator-framework/operator-sdk-samples/memcached-operator/pkg/controller/memcached: imported or required, but missing from Gopkg.lock's input-imports

# vendor is out of sync:
github.com/operator-framework/operator-sdk-samples: unused project
camilamacedo@Camilas-MacBook-Pro ~/go/src/operator-sdk-samples/memcached-operator (ISSUE_1464) $ dep version
dep:
 version     : v0.5.4
 build date  : 2019-06-14
 git hash    : 1f7c19e
 go version  : go1.12.6
 go compiler : gc
 platform    : darwin/amd64
 features    : ImportDuringSolve=false
camilamacedo@Camilas-MacBook-Pro ~/go/src/operator-sdk-samples/memcached-operator (ISSUE_1464) $ 
```

Reply:
@LiliC I am closing this comment. I will try to figure out why. if you have any idea please let me know :-)

Reply:
@camilamacedo86 that is weird, but I have never seen this before. But it says `github.com/operator-framework/operator-sdk-samples: unused project` so maybe remove the vendor directory locally and do dep ensure again? It does not complain at all for me, does `go build ...` work for you without the samples repo as part of the Gopkg.lock file?